apps/dashboard/src/pages/api-keys.tsx

```diff
@@ -0,0 +1,149 @@
+                        </InputField>
+                        <Button
+                          variant="outline"
+                          size="icon"
+                          onClick={toggleApiKeyVisibility}
+                          aria-label={showApiKey ? 'Hide API Key' : 'Show API Key'}
+                        >
+                          {showApiKey ? <RiEyeOffLine className="size-4" /> : <RiEyeLine className="size-4" />}
+                        </Button>
+                        <CopyButton valueToCopy={form.getValues('apiKey')} />
+                      </div>
+                      <p className="text-foreground-600 text-xs">
+                        Use this key to authenticate your API requests. Keep it secure and never share it publicly.
+                      </p>
+                    </div>
+
+                    <div className="space-y-2">
+                      <label className="text-foreground-600 text-sm font-medium">Application Identifier</label>
+                      <div className="flex items-center gap-2">
+                        <InputField className="flex-1">
+                          <input
+                            readOnly
+                            value={form.getValues('identifier')}
+                            className="flex-1 bg-transparent outline-none"
+                          />
+                        </InputField>
+                        <CopyButton valueToCopy={form.getValues('identifier')} />
+                      </div>
+                      <p className="text-foreground-600 text-xs">
+                        The public key identifier that can be exposed to the client applications, and used to
+                        authenticate UI Components
+                      </p>
+                    </div>
+
+                    <div className="space-y-2">
+                      <label className="text-foreground-600 text-sm font-medium">Environment ID</label>
+                      <div className="flex items-center gap-2">
+                        <InputField className="flex-1">
+                          <input
+                            readOnly
+                            value={form.getValues('environmentId')}
+                            className="flex-1 bg-transparent outline-none"
+                          />
+                        </InputField>
+                        <CopyButton valueToCopy={form.getValues('environmentId')} />
+                      </div>
+                    </div>
+                  </div>
+                </CardContent>
+              </Card>
+            </Form>
+            <div className="column flex gap-2 p-6 pt-0">
+              <div className="flex flex-col gap-2">
+                <RiLockStarFill className="h-10 w-10" />
+                <h2 className="text-foreground-950 font-medium">Environment Keys</h2>
+                <p className="text-foreground-400 text-xs">Copy and manage your public and private keys</p>
+                <a
```

Comment:
I suggest introducing an ExternalLink primitive for these links and making the font a little bit smaller (1 or 2 px) as they feel to big right now compared to the rest of the typography.